frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -422,8 +422,9 @@ function VirtualKey:onHoldSelect()
 end
 
 function VirtualKey:onSwipeKey(arg, ges)
+    if G_reader_settings:isFalse("keyboard_swipes_enabled") then return true end
     Device:performHapticFeedback("KEYBOARD_TAP")
-    if self.flash_keyboard and not self.skipswipe then
```

Comment:
Iirc that's intended for internal use (like on the shift key), regardless whether it's fully hooked up at this time. Basically see https://github.com/koreader/koreader/blob/58cb1ff42b522011b98c4bccd114ce11d24eaf9d/frontend/ui/widget/virtualkeyboard.lua#L62-L64

Reply:
But removing it is probably fine given it hasn't served any purpose the past x years.